homeassistant/components/knx/expose.py

```diff
@@ -101,9 +101,7 @@ def _init_expose_state(self) -> None:
         """Initialize state of the exposure."""
         init_state = self.hass.states.get(self.entity_id)
         init_value = self._get_expose_value(init_state)
-        self.device.sensor_value.value = (
-            init_value if init_value is not None else self.expose_default
-        )
+        self.device.sensor_value.value = init_value
```

Comment:
```suggestion
        # delete the previous line
        self.device.sensor_value.value = self._get_expose_value(init_state)
```
We don't really need `init_value` anymore then.

Reply:
it's better for debugging, isn't it? But I do whatever you prefer

Reply:
This was more a concern of readability. Not sure about debugging, I have never used pdb 🙈 but I guess it doesn't matter.